src/pages/candidate-details/candidate-details.ts

```diff
@@ -50,13 +56,29 @@ export class CandidateDetailsPage extends BasePageComponent implements OnInit, O
   ) {
     super(platform, navController, authenticationProvider);
     this.slotId = navParams.get('slotId');
+    this.slotChanged = navParams.get('slotChanged');
   }
 
   ngOnInit(): void {
-    
-    this.store$.dispatch(new ClearChangedSlot(this.slotId));
-
     this.pageState = {
+      specialNeeds$: this.store$.pipe(
+        select(getJournalState),
+        select(getSlots),
```

Comment:
In journal page, we only use `select` for the very top level of the state. I'm not against multiple selects if it works, but probably worth a wider discussion with the team about what convention we want to use.